CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(SuperiorMySqlpp VERSION 0.6.1)
```

Comment:
Hardcoded version looks like a red flag to me. This requires a TODO comment at very least, though  changelog parsing solution is preferable. We must consider if/how we're going to change debian changelog first, of course.

Reply:
Added TODO for now